src/test/java/GoRestTest.java

```diff
@@ -0,0 +1,119 @@
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.http.Header;
+import io.restassured.response.Response;
+import io.restassured.response.ValidatableResponse;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+public class GoRestTest {
+
+    private final String token = "a27efba58da81f96b92922bc4ed805103264a09f6819ac931005cd609f7419fa";
+
+    private final String body_template = """
+            {
+                "name": "Shake:%s",
+                "email": "ShakeGh_11%s@epam.com",
+                "gender": "Female",
+                "status": "Inactive"
+            }
+            """;
+
+    Header header = new Header("Authorization", "Bearer " + token);
+
+    @BeforeClass
+    public void setUp() {
+        RestAssured.baseURI = "https://gorest.co.in/";
+        RestAssured.basePath = "public-api/";
+    }
+
+    @Test
+    public void testPost() {
+
+        final long unique_id = System.currentTimeMillis();
+        String body = String.format(body_template, unique_id, unique_id);
+
+        ValidatableResponse responsePost = RestAssured
+                .given()
+                .header(header)
+                .contentType(ContentType.JSON)
+                .body(body)
+                .when()
+                .post("users")
+                .then();
+
+        SoftAssert softAssert = new SoftAssert();
+        softAssert.assertEquals(Integer.parseInt(getJsonPathParam(responsePost, "code")), 201, "User was not created.");
+        softAssert.assertEquals(getJsonPathParam(responsePost, "data.name"), String.format("Shake%s", unique_id), "User was not created.");
+    }
+
+    @Test
+    public void testDuplicate() {
+
+        final long unique_id = System.currentTimeMillis();
```

Comment:
please follow java naming convention, it should be
```suggestion
        final long uniqueId = System.currentTimeMillis();
```
also on line 54 and 86